addons/editor-dark-mode/addon.json

```
@@ -1014,6 +1011,7 @@
         "primary": "#4d97ff",
         "highlightText": "#4d97ff",
         "menuBar": "#2f4066",
+        "popup": "#333a",
```

```suggestion
        "popup": "#000a",
```
I think transparent black looks better than dark gray - especially in this preset because it mostly uses shades of blue.